resources/ext/ezbake.conf

```diff
@@ -21,7 +21,8 @@ ezbake: {
                          "mkdir -p /etc/puppet/manifests"],
                install: ["echo \\\"os-settings: {\\\"                         > $DESTDIR/$projconfdir/conf.d/os-settings.conf",
                          "echo \\\"    ruby-load-path: [$rubylibdir]\\\"      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ]
+                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ],
+               pre-start-action: ["install --group={{user}} --owner={{user}} -d /var/run/puppet"]
```

Comment:
Will this action execute every time the service starts, or only on package installation?  If it doesn't execute every time the service starts it might be better to move out of packaging and into the service management behavior, e.g. the init or unit file script.